chapter_linear-classification/softmax-regression-concise.md

```diff
@@ -70,6 +70,14 @@ We use a `Flatten` layer to convert the 4th order tensor `X`
 by keeping the dimension along the first axis unchanged.
 :end_tab:
 
+:begin_tab:`jax`
+Flax allows users to write the network class in a more compact way
+using `@nn.compact` dectorator. With `@nn.compact`, one
+can simply write all network logic inside a single “forward pass”
+method, without needing to define the standard `setup` method in
```

Comment:
What is the `setup` method？

Reply:
https://github.com/d2l-ai/d2l-en/blob/36dccad4c5c0793974028316fc7ded2ee5151920/d2l/jax.py#L192

Reply:
It's far away and it's hard to refer to code comments (not prominent) in previous sections. Can we add more descriptions of the setup method here?